examples/timestamp/signed_timestamp.c

```diff
@@ -36,67 +36,48 @@
 /* --- BEGIN TPM Timestamp Test -- */
 /******************************************************************************/
 
-typedef struct tpmKey {
-    TPM_HANDLE          handle;
-    TPM2B_AUTH          auth;
-    TPM2B_PRIVATE       priv;
-    TPM2B_PUBLIC        pub;
-    TPM2B_NAME          name;
-} TpmKey; /* Type used to store the output from Key generation */
-
-
 int TPM2_Timestamp_Test(void* userCtx)
 {
     int rc;
-    TPM2_CTX tpm2Ctx;
+    WOLFTPM2_DEV dev;
     TPMS_TIME_ATTEST_INFO* attestedTime = NULL;
 
     union {
-        GetTime_In getTime;
-        /* For creating keys */
-        CreatePrimary_In createPri;
-        Create_In create;
         /* For managing TPM session */
         StartAuthSession_In authSes;
         PolicySecret_In policySecret;
-        /* Loading a key and removing objects */
-        Load_In load;
+        /* For removing keys after use */
         FlushContext_In flushCtx;
         byte maxInput[MAX_COMMAND_SIZE];
     } cmdIn;
     union {
         ReadClock_Out readClock;
         GetTime_Out getTime;
-        /* Output from creating keys */
-        CreatePrimary_Out createPri;
-        Create_Out create;
         /* Output from session operations */
         StartAuthSession_Out authSes;
         PolicySecret_Out policySecret;
-        /* Output from loading a key into the TPM */
-        Load_Out load;
         byte maxOutput[MAX_RESPONSE_SIZE];
     } cmdOut;
 
     TPM_HANDLE sessionHandle = TPM_RH_NULL;
 
-    TpmKey endorse = { .handle = TPM_RH_NULL }; /* EK  */
-    TpmKey storage = { .handle = TPM_RH_NULL }; /* SRK */
-    TpmKey rsaKey = { .handle = TPM_RH_NULL };  /* AIK */
+    WOLFTPM2_KEY endorse = { .handle.hndl = TPM_RH_NULL }; /* EK  */
```

Comment:
An XMEMSET works too and ensures entire key is cleared.

Reply:
I followed the pattern used in native_tests, where we have handles explicitly set. Are you saying to use XMEMSET on all handles before wolfTPM2_Init()?

Reply:
Your named init to TPM_RH_NULL works, but to ensure best forward compatiblity with code the memset is best. The memset should be done early, so any failure will know the handle hasn't been setup when `wolfTPM2_UnloadHandle` is called in the failure case at the end.

Reply:
Indeed. Before we init just a handle, now we have a whole structure that has more members and sub(member) structures.